Added the MapRemoteFloret (#220)
cauliframework/cauli: docs/guides/Florets.md

## FindReplaceFloret

The [FindReplaceFloret](https://cauli.works/docs/Classes/FindReplaceFloret.html) can be used to easily replace any part of a request or response by using a [RecordModifier](https://cauli.works/docs/Classes/FindReplaceFloret/RecordModifier.html)

## MapRemoteFloret

> The [MapRemoteFloret](https://cauli.works/docs/Classes/MapRemoteFloret.html) can modify the url of a request before the request is performed. This is esp. helpful to change the app to use a staging or testing server.

Reviewer: Same as the source code documentation
```suggestion
The [MapRemoteFloret](https://cauli.works/docs/Classes/MapRemoteFloret.html) can modify the url before the request is performed. This is esp. helpful when using a staging or testing server.
```
